.golangci.yml

```diff
@@ -17,6 +17,8 @@ linters:
     - nlreturn
     # Linter requires periods at end of comments
     - godot
+    # Go linter that checks package imports are in a list of acceptable packages
+    - depguard
```

Comment:
There must have been a recent change to golangci-lint that requires this? I just added to the ignore list because it was complaining about importing "github.com/goreapers/gograveyard" in main.go

Reply:
agreed